doc/classes/AnimationNode.xml

```diff
@@ -80,12 +80,12 @@
 			<param index="0" name="animation" type="StringName" />
 			<param index="1" name="time" type="float" />
 			<param index="2" name="delta" type="float" />
-			<param index="3" name="seeked" type="bool" />
```

Comment:
This is not correct, `seeked` is a term used across the codebase

Reply:
All right. Reverting ...

Reply:
The fact it's used across codebase doesn't change the fact that it's not proper, grammatical English and typo-catchers WILL point this out

Reply:
Exactly why sweeping typo fixes isn't a useful tool for such

Further it isn't necessarily un-grammatical, "seek" in this case isn't the same as "to seek" necessarily, take "computer mouse" , "computer mouses" for example

Reply:
If it isn't "to seek" it should be noted somewhere (e.g. where it's first used) as it's bound to confuse both non-native English speakers delving into Godot code AND typo catchers

Reply:
In fact this is established computer terminology:
https://en.wiktionary.org/wiki/seeked

For over 30 years now

Reply:
Wow, TIL! Link worth putting in a comment somewhere so that other non-natives don't get confused

Reply:
I don't think that would help much, you would have to find that one specifically, and you couldn't do it to the docs